modules/preview-react/table/stories/Tables.stories.mdx

```diff
@@ -0,0 +1,96 @@
+import {Table} from '@workday/canvas-kit-preview-react/table';
+// examples
+import {Basic} from './examples/Basic';
+import {BasicWithHeading} from './examples/BasicWithHeading';
+import {FixedColumn} from './examples/FixedColumn.tsx';
+
+<Meta title="Preview/Table" component={Table} />
+
+# Canvas Kit Table
+
+`Table` is a simple styled compound component that renders a
+[table](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table) element. It is used to
+present information in a two-dimensional table comprised of rows and columns of cells containing
+data.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-preview-react
+```
+
+## Usage
+
+Each piece of the compound component renders an HTML element that is part of a table element:
+
+- `Table`: Renders [table](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table)
+- `Table.Caption`: Renders
+  [caption](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption)
+- `Table.Head`: Renders [thead](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead)
+- `Table.Body`: Renders [tbody](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tbody)
```

Comment:
The `SymbolDoc` component from Nicholas' recent docgen work will handle a lot of this documentation in your Usage section automatically (i.e., which HTML elements each of the Table components renders and even which layout component serves as the foundation for each of those Table components).

You'll need to make the following changes to your stories MDX:
```
// Top of Table.stories.mdx
import {SymbolDoc} from '@workday/canvas-kit-docs';

...

// Bottom of Table.stories.mdx
## Component API

<SymbolDoc name="Table" fileName="/preview-react/" />
```

It's customary to include the `## Component API` section after the `## Usage` section. See any of our other stories.mdx files for an example (such as [Modal.stories.mdx](https://github.com/Workday/canvas-kit/blame/master/modules/react/modal/stories/Modal.stories.mdx)).

Reply:
Updated